deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -502,3 +502,15 @@ impl From<ChatItem> for JSONRPCMessageListItem {
         }
     }
 }
+
+#[derive(Deserialize, TypeDef)]
+#[serde(rename_all = "camelCase")]
+pub struct DraftMessage {
```

Comment:
naming: this is more a message builder struct than a "draft", we already have drafts in core, the therm already has a meaning so I would not reuse the term draft here

Reply:
MessageOptions? MessageData?

Reply:
It is used in generated typings

Reply:
I named it OutgoingMessage initially, but that is also a concept in delta chat, you have outgoing and incoming messages so it was a bit confusing, I struggled to find a proper name, I am aware draft was not great either but in which situation would you return a "DraftMessage" in this API? you already have a MessageObject, so I think for the API it is fine to say the parameter is a drafted message to be sent, I am ok with other names you come with and that can be tweaked in a follow up PR by anyone, I need this stuff quickly, don't want to stumble with naming too much myself if it can be avoided :sweat_smile:

Reply:
other name I though was "NewMessage" but that also looks a bit problematic, maybe "MessageProperties" ??

Reply:
for now renamed DraftMessage to MessageData as it was one of the suggested names, feel free to further tweak in a follow up PR